### DISCUSSION TITLE
Google Fonts

### DISCUSSION THREAD

**Discussion by shinanqu** [oleeskild/obsidian-digital-garden#196] [Q&A] [answered]
In Germany, we always try to do things a bit more complicated. While all over the world you can just create a homepage and enjoy a happy life, in Germany on the other side you are not allowed to use Google Fonts from American servers. You can, however, download these fonts, upload them on your own homepage and thus no hidden access to American servers is needed. It's a huge huge pain for every website owner in Germany :-(

And it also means, strictly speaking, that many website generators are not allowed to be used in Germany. As long as you cannot host Google Fonts locally or don't use them at all, you always run the risk to receive a random letter from a random lawyer, asking you for money.

How about Digital Garden. Is there any (simple) way to host the fonts locally or avoid Google Fonts? Or does it all relate to the themes and thus I would have to find a theme which doesn't use Google Fonts?

Thank you!

**Comment by uroybd** [comment, marked as answer]
It is very much possible. Especially with this feature https://dg-docs.ole.dev/advanced/adding-custom-components/

**Reply by shinanqu** [reply]
I will check it out!
I don't want anything fancy. The principle of a Digital Garten is to make it very simple (compared with Wordpress etc.), I guess. But at least make it legal in Germany as well would be a nice feature as well.

**Reply by uroybd** [reply]
Sorry, I kinda overlooked that the template already adds Roboto. Anyway, I added a setting to serve it locally or from CDN here #202 

Once @oleeskild merges this, you can just use the toggle and be all set.

**Reply by shinanqu** [reply]
> Sorry, I kinda overlooked that the template already adds Roboto. Anyway, I added a setting to serve it locally or from CDN here #202
> 
> Once @oleeskild merges this, you can just use the toggle and be all set.

Oh gosh that so amazing. It is really so extremely helpful for all Germans! Finally, we can post text online without being scared all the time. Thank you so much!
